### PR TITLE
Minor edits to file structure page

### PR DESCRIPTION
**Pull request by jbogarthyde**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/file-structure.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ The top level of the workspace contains a number of workspace-wide configuration
 | :--------------------- | :------------------------------------------|
 | `.editorconfig`        | Configuration for code editors. See [EditorConfig](https://editorconfig.org/). |
 | `.gitignore`           | Specifies intentionally untracked files that [Git](https://git-scm.com/) should ignore. |
-| `angular.json`        | CLI configuration for all projects in the workspace, including configuration options for build, serve, and test tools that the CLI uses, such as [Karma](https://karma-runner.github.io/) and [Protractor](http://www.protractortest.org/).  |
+| `angular.json`         | CLI configuration defaults for all projects in the workspace, including configuration options for build, serve, and test tools that the CLI uses, such as [TSLint](https://palantir.github.io/tslint/), [Karma](https://karma-runner.github.io/) and [Protractor](http://www.protractortest.org/).  |
```
Comment (jenniferfell) [inline aio/content/guide/file-structure.md]: Oxford comma: , and Protractor

File: aio/content/guide/file-structure.md [hunk 2]
```diff
@@ -20,12 +20,12 @@ The top level of the workspace contains a number of workspace-wide configuration
...
 | `node_modules`         | Provides [npm packages](guide/npm-packages) to the entire workspace. |
-| `package.json`        | Lists package dependencies. See [npm documentation](https://docs.npmjs.com/files/package.json) for the specific format and contents of this file.|
-| `tsconfig.app.json`   | Default [TypeScript](https://www.typescriptlang.org/) configuration for apps in the workspace. |
-| `tsconfig.spec.json`  | Default TypeScript configuration for e2e test apps in the workspace. |
-| `tslint.json`         | Default [TSLint](https://palantir.github.io/tslint/) configuration for apps in the workspace. |
+| `package.json`         | Configures [npm package](guide/npm-packages) dependencies that are available to all projects in the workspace. See [npm documentation](https://docs.npmjs.com/files/package.json) for the specific format and contents of this file. |
```
Comment (jenniferfell) [inline aio/content/guide/file-structure.md]: A nit, but the title now has "dependencies" in it, so maybe this: [npm package dependencies](guide/npm-packages). 

Trivia: All dependencies are packages. All packages listed there are dependencies. So "package dependencies" is a bit redundant, but I think it's a useful combo phrase here.

File: aio/content/guide/file-structure.md [hunk 3]
```diff
@@ -20,12 +20,12 @@ The top level of the workspace contains a number of workspace-wide configuration
...
+| `tsconfig.app.json`    | Default [TypeScript](https://www.typescriptlang.org/) configuration for apps in the workspace. |
```
Comment (jenniferfell) [inline aio/content/guide/file-structure.md]: I don't see this at the workspace level. I see the tsconfig.*.json files in my project folders. At the workspace level I see tsconfig.json and tslint.json. The description matches, but I think the filenames at workspace level don't have the middle parts.

Comment (jenniferfell) [inline aio/content/guide/file-structure.md]: I'm not sure how/if to add this, but it also contains any Angular template compiler options. In my initial workspace and apps, I don't see any Angular compiler options, but this is where developers add them. 

Maybe: Default TypeScript configuration for apps in the workspace, including TypeScript and Angular template compiler options. 

Related link for TS config in general: https://angular.io/guide/typescript-configuration

We're moving the options to their own file, but we'll link from that TS config guide. For context for you, options are here now: https://angular.io/guide/aot-compiler#angular-template-compiler-options

File: aio/content/guide/file-structure.md [hunk 4]
```diff
@@ -20,12 +20,12 @@ The top level of the workspace contains a number of workspace-wide configuration
...
+| `tsconfig.spec.json`   | Default TypeScript configuration for e2e test apps in the workspace. |
```
Comment (jenniferfell) [inline aio/content/guide/file-structure.md]: See note above re what I see in a new workspace. I only see this file in project folders.

**Review by jenniferfell — CHANGES_REQUESTED** [review]
Re tsconfig.*.json: I don't see these at the workspace level when I do ng new, but can developers choose to add them there manually?

File: aio/content/guide/file-structure.md (hunk 3 above)
Comment (jbogarthyde) [reply aio/content/guide/file-structure.md]: @IgorMinar -- how is this supposed to be? Are there separate app and spec tsconfig files? Are they generated automatically? Are they named like this? (I'm not sure anymore where this info came from originally.)

**Comment by jbogarthyde** [comment]
@jenniferfell Your requested changes were incorporated - please remove blocker.

**Review by jenniferfell — APPROVED** [review]
LGTM

**Pull request closed without merging**